components/Card.js

```diff
@@ -1,46 +1,25 @@
 import styled from "styled-components";
 import { useState } from "react";
 
-export default function Card({ tasks }) {
-  const [toggleCardsize, setToggleCardsize] = useState(false);
+export default function Card({ name, note, time }) {
+  const [toggleDetailsCard, setToggleDetailsCard] = useState(false);
   function handleClick() {
-    setToggleCardsize(!toggleCardsize);
-  }
-  function handleSubmit(event) {
-    event.preventDefault();
+    setToggleDetailsCard(!toggleDetailsCard);
   }
   return (
-    <StyledUl>
-      {tasks.map((task) => (
-        <StyledCard key={task.id}>
-          <h2>{task.name}</h2>
-          <button onClick={handleClick}>details</button>
-          {toggleCardsize ? (
-            <>
-              <h3>Notes:</h3>
-              <p>{task.note}</p>
-              <h4>Estimated time:</h4>
-              <p>{task.time} minutes</p>
-            </>
-          ) : (
-            ""
-          )}
-        </StyledCard>
-      ))}
-    </StyledUl>
+    <>
+      <h1>{name}</h1>
+      <button onClick={handleClick}>show details</button>
+      {toggleDetailsCard ? (
+        <>
+          <h2>Notes:</h2>
+          <p>{note}</p>
+          <h4>Estimated time:</h4>
+          <p>{time} minutes</p>
+        </>
+      ) : (
+        ""
+      )}
```

Comment:
when you show the details, the button should show "hide details" because it doesn't make sense to click on "show details" again to hide them 😄